cfpb/consumerfinance.gov
# RAF tool: dropdown and notification fixes

Fixes a few small issues with the RAF tool before launch tomorrow morning.

## Changes

- Updates raft tool js


## How to test this PR

1. Check out branch and run migrations & gulp copy
2. Create a browse page and add RAF tool block
3. Verify that tribe select options are in alphabetical order
4. Select `American Samoa` in state/territory dropdown and verify that there are no periods after links in no results notification message


## Screenshots

<img width="655" alt="Screen Shot 2021-07-27 at 1 45 41 PM" src="https://user-images.githubusercontent.com/778171/127225696-4d6f2566-9728-4263-8895-7e49e9d89761.png">

<img width="708" alt="Screen Shot 2021-07-27 at 1 45 56 PM" src="https://user-images.githubusercontent.com/778171/127225677-53e9e0f0-fa97-4592-a0bd-bb7157cc9bda.png">


## Notes and todos

- Once this app is imported as an npm module, we will link to updates in external app to show release changes
